hack/logging-dump.sh

```diff
@@ -211,16 +213,6 @@ check_curator_connectivity() {
   oc exec $pod -- curl -ILvs --key /etc/curator/keys/key --cert /etc/curator/keys/cert --cacert /etc/curator/keys/ca -XGET https://$es_host:$es_port &>> $curator_folder/$pod
```

Comment:
not sure what we can do about check_curator - we can't use any pod based tests since there will almost never be a pod running

Reply:
i think we may be able to spin up a debug pod...

Reply:
Skipping for now unless I hear otherwise from the workloads team: 
```
$ oc debug cronjob/curator -- curl -ILvs --key /etc/curator/keys/key --cert /etc/curator/keys/cert --cacert /etc/curator/keys/ca -XGET https://logging-es:9200
error: cannot debug curator: unable to extract pod template from type *v1beta1.CronJob
```

Reply:
it was my oc version...